cgltf_write.h

```diff
@@ -937,10 +937,7 @@ cgltf_size cgltf_write(const cgltf_options* options, char* buffer, cgltf_size si
 		cgltf_write_line(context, "]");
 	}
 
-	if (data->scene)
-	{
-		cgltf_write_intprop(context, "scene", data->scene - data->scenes, -1);
-	}
+	CGLTF_WRITE_IDXPROP("scene", data->scene, data->scenes);
```

Comment:
Would this not cause problems now if don't have a default scene?

Reply:
I don't think so, unless I have missed something. `CGLTF_WRITE_IDXPROP(label, val, start)` checks if `val` is `null`, in which case nothing is written at all. This is used as a pattern throughout the code for non-mandatory properties.

Reply:
Sorry, my mistake.